selfdrive/manager/test/test_manager.py

```diff
@@ -20,6 +21,10 @@ def setUp(self):
     os.environ['PASSIVE'] = '0'
     HARDWARE.set_power_save(False)
 
+    # ensure clean CarParams
+    params = Params()
+    params.remove("CarParams")
```

Comment:
other params can also be in a bad state
```suggestion
    # ensure clean params
    params = Params()
    params.clear_all()
```